src/parse.c

```diff
@@ -2807,6 +2807,8 @@ static const mapping_entry_handler dhcp6_overrides_handlers[] = {
     {"renderer", YAML_SCALAR_NODE, {.generic=handle_netdef_renderer}, NULL}, \
     {"routes", YAML_SEQUENCE_NODE, {.generic=handle_routes}, NULL}, \
     {"routing-policy", YAML_SEQUENCE_NODE, {.generic=handle_ip_rules}, NULL}, \
+    {"hairpin", YAML_SCALAR_NODE, {.generic=handle_netdef_tristate}, netdef_offset(bridge_hairpin)}, \
+    {"learning", YAML_SCALAR_NODE, {.generic=handle_netdef_tristate}, netdef_offset(bridge_learning)}, \
```

Comment:
```suggestion
    {"mac-learning", YAML_SCALAR_NODE, {.generic=handle_netdef_tristate}, netdef_offset(bridge_learning)}, \
```

todo: I feel like we should call this mac-learning, to stay in line with the corresponding VXLAN setting of similar functionality.